django/utils/autoreload.py

```diff
@@ -111,7 +111,9 @@ def iter_modules_and_files(modules, extra_files):
         # During debugging (with PyDev) the 'typing.io' and 'typing.re' objects
         # are added to sys.modules, however they are types not modules and so
         # cause issues here.
-        if not isinstance(module, ModuleType) or module.__spec__ is None:
+        if (not isinstance(module, ModuleType) or
+                not hasattr(module, '__spec__') or
+                module.__spec__ is None):
```

Comment:
`getattr(module, '__spec__', None) is None` is more concise.

Reply:
You can drop the `or module.__spec__ is None`, both are now contained in the `module.__spec__ is None` check.

Reply:
Right, sorry, I should have seen that...